tests/data/condor_submit_dag/simple.cmd

```diff
@@ -0,0 +1,96 @@
+
+# generated by jobsub_lite
+# debug
+universe           = vanilla
+executable         = simple.sh
+arguments          = --find_setups --prescript-unquote echo%20JOBSUBJOBSECTION%20is%20%24%7BJOBSUBJOBSECTION%7D --setup-unquote hypotcode%20v1_1 --setup-unquote ifdhc%20v2_6_10%2C%20ifdhc_config%20v2_6_15 --self_destruct_timer 700 --debug --getconfig --limit 1 --appvers v1_1 --metadata_extractor hypot_metadata_extractor --addoutput gen.troot --rename unique --dest /pnfs/fermilab/users/mengel/dropbox --add_location --declare_metadata --addoutput1 hist_gen.troot --rename1 unique --dest1 /pnfs/fermilab/users/mengel/dropbox --add_location1 --declare_metadata1 --exe hypot.exe -- -o gen.troot -c hist_gen.troot
```

Comment:
Will these tests not work if they don't write out to your `/pnfs/fermilab` area?

Reply:
This appears to be fixed now.